### ISSUE TITLE
Graphical and color distortions in The Witcher

### ISSUE THREAD

**Issue by shmerl** [doitsujin/dxvk#4178]
I noticed that quite for a long time already in some scenes in the Witcher there are glitches when using dxvk.

Here is a small video example (see Shani's hair and some general flickering and light anomalies especially around that candle on the table):

[test.webm](https://github.com/user-attachments/assets/f88adb34-a36e-465f-bd5c-a8719c08b298)

Save file:

[witcher_save.zip](https://github.com/user-attachments/files/16435519/witcher_save.zip)

Set up:

* Game:  The Witcher Enhanced Edition, latest GOG release.
* Wine: 9.14 + esync
* dxvk: git-1077d6a
* GPU: AMD Sapphire Nitro+ RX 7900 XTX
* Kernel/amdgpu: 6.10.2
* Mesa/radv: git-deb1d1f9c3

Please let me know if some kind of Vulkan captures could help narrow this down, I'll try to add more info if needed.

**Comment by Blisto91** [comment]
I took a quick poke and this seems to be a d3dx9 issue. If you use the Windows native version of `d3dx9_35` it should go away.

**Comment by shmerl** [comment]
Yeah, just tested it and can confirm that native `d3dx9_35.dll` works around it. I can file a Wine bug about it I hope they can accept it even though it's exposed by dxvk use case.

**Comment by shmerl** [comment]
Opened https://bugs.winehq.org/show_bug.cgi?id=57022

**Comment by Blisto91** [comment]
Closing since this is tracked on the Wine tracker
